glossary.md

```diff
@@ -1,7 +1,55 @@
 ## Glossary
 
-* Username
-* Password
-* Session & Session Location
-* Visits
-* Vitals
+**administrative staff**: Individuals who manage people or data in a clinical setting.
+
+**android client**: The OpenMRS mobile client for accessing OpenMRS server instance on Android devices.
+
+**chart**: A graph for patients, measuring height, weight and other observations corresponding to recorded patient vitals.
+
+**clinician**: A doctor, nurse, or other clinical officer who provides health care to patients.
+
+**concept**: The idea that encompasses any question which can be asked about a patient; an observable point of data.
+
+**data**: A piece of knowledge that can be reduced to a single value.
+
+**electronic medical record (EMR)**: A computer system that allows for recording, storage, and retrieval of information related to the delivery of health care to patients.
+
+**encounter**: A clinical transaction between a patient and one or more healthcare providers for the purpose of providing patient services or assessing the health status of the patient. An encounter happens at one point in time. [http://www.astm.org/Standards/E1384.htm](http://www.astm.org/Standards/E1384.htm)
```

Comment:
What is the purpose of the link in this definition?

Reply:
To give a more elaborate description on what  an encounter is. I notice now it is withdrawn in March 2017. I'll remove this link.